.github/workflows/build-package.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone -b v0.8.10 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b v0.8.12 --depth=1  https://github.com/citusdata/tools.git tools
```

Comment:
Will move the tag into develop branch after tools PR https://github.com/citusdata/tools/pull/216 merge